mcs/class/corlib/Mono/Runtime.cs

```diff
@@ -37,6 +37,13 @@ namespace Mono {
 #endif
 	static class Runtime
 	{
+		/*
```

Comment:
It'd be better to have it in a new class as this class will probably be fully excluded, what about `RuntimeInformation` or something like that

Reply:
We tell people to check for this class to see if they run on mono, how can it be excluded ?

Reply:
I think exactly because of that, it won't be Mono as they expect it

Reply:
Mono.RuntimeStructs ?

Reply:
@vargaz I am fine with moving the version info to Mono.RuntimeStructs